cmd/mungedocs/analytics_test.go

```diff
@@ -27,53 +27,50 @@ func TestAnalytics(t *testing.T) {
 		in  string
 		out string
 	}{
-		{`aoeu`, `aoeu
-
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`},
-		{`aoeu
-
-
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-`, `aoeu
-
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`},
-		{`aoeu
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`, `aoeu
-
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`},
-		{`aoeu
-
-
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-
-
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`, `aoeu
-
-
-` + beginMungeTag("GENERATED_ANALYTICS") + `
-[![Analytics](https://kubernetes-site.appspot.com/UA-36037335-10/GitHub/path/to/file-name.md?pixel)]()
-` + endMungeTag("GENERATED_ANALYTICS") + `
-`},
+		{"aoeu", "aoeu" + "\n" + "\n" + "\n" +
```

Comment:
Can you insert a newline after the { so that the indentation will be consistent between the two strings? It's very hard to read right now.